src/univariate/continuous/chernoff.jl

```diff
@@ -0,0 +1,317 @@
+## chernoff.jl
+##
+## The code below is intended to go with the Distributions package of Julia.
+## It was written by Joris Pinkse, joris@psu.edu, on December 12, 2017.  Caveat emptor.
+##
+## It computes pdf, cdf, moments, quantiles, and random numbers for the Chernoff distribution.
+## 
+## Most of the symbols have the same meaning as in the Groeneboom and Wellner paper.
+##
+## Random numbers are drawn using a Ziggurat algorithm.  To obtain draws in the tails, the
+## algorithm reverts to quantiles, which is slow.
+"""
+    Chernoff()
+
+The *Chernoff distribution* is the distribution of the random variable
+```math
+\\argmax_{t \\in (-\\infty,\\infty)} ( G(t) - t^2 ),
+```
+where ``G`` is standard two--sided Brownian motion.  
+
+The distribution arises as the limit distribution of various cube-root-n consistent estimators,
+including the isotonic regression estimator of Brunk, the isotonic density estimator of Grenander,
+the least median of squares estimator of Rousseeuw, and the maximum score estimator of Manski. 
+
+For theoretical results, see e.g. Kim and Pollard, Annals of Statistics, 1990.  The code for the 
+computation of pdf and cdf is based on the algorithm described in Groeneboom and Wellner, 
+Journal of Computational and Graphical Statistics, 2001.
+
+```julia
+Chernoff()
+pdf(Chernoff(),x::Real)
+cdf(Chernoff(),x::Real)
+logpdf(Chernoff(),x::Real)
+survivor(Chernoff(),x::Real)
+mean(Chernoff())
+var(Chernoff())
+skewness(Chernoff())
+kurtosis(Chernoff())                  
+kurtosis(Chernoff(), excess::Bool)
+mode(Chernoff())
+entropy(Chernoff())
+rand(Chernoff())
+rand(rng, Chernoff()
+cdf(Chernoff(),-x)              #For tail probabilities, use this instead of 1-cdf(Chernoff(),x) 
+```
+"""
+struct Chernoff <: ContinuousUnivariateDistribution
+end
+
+# The following arrays of constants have been precomputed to speed up computation.
+# The first two correspond to the arrays a and b in the Groeneboom and Wellner article.
+# The array az contains roots of the airyai functions (atilde in the paper).
```

Comment:
Array names out of sync

Reply:
@matbesancon suggested these changes

Reply:
@eulerkochy there are still arrays `a` and `b` which would need explicit names, along with the functions `p`, `g` defined below

Reply:
Ok, on it!

Reply:
@matbesancon the functions `p` and `g` have direct reference to the [paper](https://www.stat.washington.edu/jaw/JAW-papers/jaw-groeneboom.jcgs.01.pdf). I don't understand what explicit names to state for them.

Reply:
![image](https://user-images.githubusercontent.com/33925317/55326918-8853ac00-54a6-11e9-8d50-8959c1ab5a03.png)
Same applies for `a` and `b`.

Reply:
yes but it's really not handy in a big package like `Distributions` to use notations which can mean different things depending on the context and distribution the function relates to. As @mschauer mentioned, one solution would be to define a submodule `ChernoffComputation` in which all these constants and intermediate functions can live

Reply:
Ok, that would be the most feasible solution!